apps/avatax/integration/data/maps/channel.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Channel": {
+    "USA": {
+      "id": "Q2hhbm5lbDoz",
+      "slug": "channel-us",
+      "deliveryMethodId": "U2hpcHBpbmdNZXRob2Q6MTY="
```

Comment:
Yes, we hardcode most of the data instead of creating each channel/product, etc in the test. The same approach we use in dashboard PW tests https://github.com/saleor/saleor-dashboard/blob/main/playwright/data/e2eTestData.ts we store all IDs in one file.

Reply:
Yes, that was intentional - we don't want to use env variables for storing IDs. Since we will be using snapshots, the IDs should remain the same across different Saleor versions. 

Off band, we've agreed that if a need arises we will add new IDs for specific Saleor versions or change our approach if our tests become too flaky.

> at least we can create some "test-vars.json" that will keep them in a single space?

Is it OK if we leave the JSONs as it is? I think the split between files for each type of data makes it more readable than putting it into one large file. It will come in handy once we add more product types, channels, etc. WDYT?

> Also, if possible, I think we should use .js instead of json, because we can add comments and structure files better

Sure, we could do that, but that would require to manually add the `maps` and `templates` into Pactum: https://pactumjs.github.io/guides/data-management.html#data-map

By using JSONs we can just use this single utility to load everything: https://pactumjs.github.io/guides/data-management.html#loading-data